### ISSUE TITLE
Simply start acting: Improve of Edit Your Profile form 

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/accessibility#56]
Hi all. This is the next idea for simple PR. Who will do it?

## Improve accessibility of Edit Your Profile form 

There are two buttons in the form:
* Submit
* Cancel.

But only one of them is actually a button - "Submit" button. The second one looks like a button, but is a link. Take a look at the source code to see for yourself.

It should be a button, not a link. 

When a keyboard user sees a button or screen reader user hears that this is a button, he wants to run it using Spacebar. Unfortunately, Spacebar does not launch the expected action. 

What do you think about it? If you agree, don't wait - start working. Prepare PR. You need to fix the file: components/com_users/tmpl/profile/edit.php.

**Comment by zwiastunsw** [comment]
Solved by @SharkyKZ: https://github.com/joomla/joomla-cms/pull/24088#issuecomment-469452356
